src/datamodeltranslator/5_1_0/sd/dataModelTranslation/dataModelTranslation_sd.tex

```diff
@@ -0,0 +1,278 @@
+As a complement to the explicitly defined types in this section, there is also a list of implicit primitive types in Section \ref{sec:model:primitives}, which are used to represent things like hashes and identifiers.
+
+\phantomsection
+\msubsection{struct}{DataModelTranslationInitRequest}
+ 
+\begin{table}[ht!]
+\begin{tabularx}{\textwidth}{| p{3.6cm} | p{4.9cm} | p{2cm} | X |} \hline
+\rowcolor{gray!33} Field & Type & Mandatory & Description \\ \hline
+accessToken & \pref{String} & no & The access token that might be required. \\ \hline
+inputModelId & \pref{DataModelID} & yes & The identifier of the data model to be translated. \\ \hline
+outputModelId & \pref{DataModelID} & yes & The identifier of the required result data model. \\ \hline
+payload & \pref{Object} & yes & The input data to be translated. \\ \hline
+settings & \hyperref[sec:model:Map]{Map}  & no & Any configuration data that might be required by a translation process. \\ \hline
+\end{tabularx}
+\end{table}
+
+\msubsection{struct}{Map}
+
+An \pref{Object} which maps \pref{String} keys to primitive, \pref{Object} or list values.
+
+\msubsection{struct}{DataModelTranslationInitResponse}
+ 
+\begin{table}[ht!]
+\begin{tabularx}{\textwidth}{| p{3cm} | p{3cm} | X |} \hline
+\rowcolor{gray!33} Field & Type      & Description \\ \hline
+status & \pref{OperationStatus} & Status of the operation. \\ \hline
+taskId & \pref{TaskID} & Identifier of the initialized translation task. \\ \hline
+\end{tabularx}
+\end{table}
+
+\msubsection{struct}{ErrorResponse}
+
+\begin{table}[ht!]
+\begin{tabularx}{\textwidth}{| p{4.25cm} | p{3.5cm} | X |} \hline
+\rowcolor{gray!33} Field & Type      & Description \\ \hline
+status & \pref{OperationStatus} & Status of the operation. \\ \hline
+errorMessage & \pref{String} & Description of the error. \\ \hline
+errorCode &\pref{Number}  & Numerical code of the error. \\ \hline
+type & \pref{ErrorType} & Type of the error. \\ \hline
+origin & \pref{String} & Origin of the error. \\ \hline
+\end{tabularx}
+\end{table}
+
+\msubsection{struct}{TaskIDRequest}
+ 
+\begin{table}[ht!]
+\begin{tabularx}{\textwidth}{| p{3.6cm} | p{4.9cm} | p{2cm} | X |} \hline
+\rowcolor{gray!33} Field & Type & Mandatory & Description \\ \hline
+accessToken & \pref{String} & no & The access token that might be required. \\ \hline
+taskId & \pref{TaskID} & yes & Identifier of the translation task. \\ \hline
+\end{tabularx}
+\end{table}
+
+\msubsection{struct}{DataModelTranslationResultResponse}
+ 
+\begin{table}[ht!]
+\begin{tabularx}{\textwidth}{| p{3cm} | p{3cm} | X |} \hline
+\rowcolor{gray!33} Field & Type      & Description \\ \hline
+operationStatus & \pref{OperationStatus} & Status of the operation. \\ \hline
+status & \pref{TaskStatus} & Status of the translation task. \\ \hline
+result & \pref{Object} & The result (translated) data payload. \\ \hline
```

Comment:
see comments about payload. It is a little more complicated because it can be a simple String if status is error.